torch/csrc/jit/python_tracer.cpp

```diff
@@ -82,13 +82,14 @@ void initPythonTracerBindings(PyObject* module_) {
       ss << *s.graph;
       return ss.str();
     })
-    .def("export", [](TracingState& s, bool verbose) {
+    .def("export", [](TracingState& s,
```

Comment:
Can we put keyword arguments on these? See http://pybind11.readthedocs.io/en/master/basics.html#keyword-args for how to do it.

Reply:
that stuff looks like black magic. I tried it before, it didn't work, and I gave up. I think there are part of pybind11 we should avoid using.